### ISSUE TITLE
Model Info

### ISSUE THREAD

**Issue by 0o120** [Anjok07/ultimatevocalremovergui#1075]
Is there some type of resource available that has information for all the models? Some type of specs and where they come from, how they were produced?

Feels like I'm mostly just blindly choosing models until I find 1 that works. Would be nice to have some resource that gives us details of the models. :)

**Comment by ChrisTG742** [comment]
Good point! I think that would be a good addition to the docs (btw, are there any "real" docs yet? I don't think so).
So maybe someone here could take the opportunity and start creating some docs for this lovely tool?

For now, one could at least check the provided links below "Credits" in the README.md.

**Comment by 0o120** [comment]
So I came across this google document. Has tons of great information. Leaving this here for anyone who might need it.

https://docs.google.com/document/d/17fjNvJzj8ZGSer7c7OFe_CNfUKbAxEh_OBv94ZdRG5c/

via this reddit comment:
https://www.reddit.com/r/audioengineering/comments/184e9kk/comment/kaut4vr/
